cfme/configure/__init__.py

```diff
@@ -0,0 +1,53 @@
+from navmazing import NavigateToSibling
+from widgetastic.widget import View
+from widgetastic_manageiq import Accordion, ManageIQTree
+from widgetastic_patternfly import Dropdown
+
+from cfme import BaseLoggedInPage
+from cfme.base import Server
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep
+
+
+class ConfigurationView(BaseLoggedInPage):
+    def in_explorer(self):
+        return self.logged_in_as_current_user
```

Comment:
This in_explorer is probably justa copypaste. Make it a proper in_configuration :)